snmp/datadog_checks/snmp/data/default_profiles/nvidia-cumulus-linux-switch.yaml

```diff
@@ -0,0 +1,241 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _generic-entity-sensor-mib.yaml
```

Comment:
Shall we move:

```
  - _base.yaml
  - _generic-if.yaml
```

to `ndvidia.yaml`

and maybe ` _generic-entity-sensor-mib.yaml` also.